packages/gatsby/src/bootstrap/load-plugins/load.js

```diff
@@ -160,11 +160,13 @@ module.exports = async (config = {}) => {
 
   // Add the auto page creator plugin
   plugins.push({
-    resolve: `gatsby-plugin-page-creator`,
+    resolve: slash(`${process.cwd()}/node_modules/gatsby-plugin-page-creator`),
```

Comment:
You'd want to use `require.resolve` here as otherwise this wouldn't work in certain situations e.g. monorepos where the package is hoisted to a higher directory.